salt/utils/network.py

```diff
@@ -206,6 +207,7 @@ def is_ip(ip):
     return is_ipv4(ip) or is_ipv6(ip)
 
 
+@jinja_filter('is_ipv4')
```

Comment:
As a side note, please do not override the existing ```is_ipv4``` filter: https://github.com/saltstack/salt/blob/develop/salt/utils/templates.py#L354 which is using the ```is_ipv4_filter``` functions.